queries/pug/highlights.scm

```diff
@@ -0,0 +1,28 @@
+(comment) @comment
+(tag_name) @tag
+(
+  (tag_name) @_tag_name
+  ; https://www.script-example.com/html-tag-liste
+  (#any-of? @_tag_name
+   "head" "title" "base" "link" "meta" "style"
+   "body" "article" "section" "nav" "aside" "h1" "h2" "h3" "h4" "h5" "h6" "hgroup" "header" "footer" "address"
+   "p" "hr" "pre" "blockquote" "ol" "ul" "menu" "li" "dl" "dt" "dd" "figure" "figcaption" "main" "div"
+   "a" "em" "strong" "small" "s" "cite" "q" "dfn" "abbr" "ruby" "rt" "rp" "data" "time" "code" "var" "samp" "kbd" "sub" "sup" "i" "b" "u" "mark" "bdi" "bdo" "span" "br" "wbr"
+   "ins" "del"
+   "picture" "source" "img" "iframe" "embed" "object" "param" "video" "audio" "track" "map" "area"
+   "table" "caption" "colgroup" "col" "tbody" "thead" "tfoot" "tr" "td" "th "
+   "form" "label" "input" "button" "select" "datalist" "optgroup" "option" "textarea" "output" "progress" "meter" "fieldset" "legend"
+   "details" "summary" "dialog"
+   "script" "noscript" "template" "slot" "canvas")
+) @constant.builtin
+(content) @none
+(quoted_attribute_value) @string
+(id) @constant
+(class) @constant
+; (attribute_name) @variable.builtin
+(attribute_name) @symbol
```

Comment:
Duplicate?

Reply:
My intention was, to make highlighting for "static" attributes different than those with Javascript interpolation.

HTML Attributes such as "v-if=" , "@click=", ":src=" are interpreted as Javascript within Vue template. That's why I used the second, more specific rule, to override the highlighting for those attribute. 
See here the "target" attribute is plain text as v-if & src are bound to Javascript values.

![Bildschirmfoto 2021-07-13 um 18 56 01](https://user-images.githubusercontent.com/147175/125493962-31caa874-4319-4204-8ab6-0c2e5ba981db.png)